src/routers/__tests__/TabRouter-test.js

```diff
@@ -604,4 +606,65 @@ describe('TabRouter', () => {
       ],
     });
   });
+
+  test(
+    'Given a StackRouter with nested TabRouter, it is possible to reset ' +
+      'the StackRouter and navigate to an arbitrary tab (BarTab) in the ' +
+      'TabRouter by supplying sub-actions to the RESET action.',
+    () => {
+      const ChildNavigator = () => <div />;
+      ChildNavigator.router = TabRouter({
+        FooTab: BareLeafRouteConfig,
+        BarTab: BareLeafRouteConfig,
+      });
+
+      const BarScreen = () => <div />;
+      const router = StackRouter({
+        BarStack: {
+          screen: BarScreen,
+        },
+        FooStack: {
+          screen: ChildNavigator,
+        },
+      });
+      const state = router.getStateForAction({ type: NavigationActions.INIT });
+      const state2 = router.getStateForAction(
+        {
+          index: 0,
+          type: NavigationActions.RESET,
+          params: { key: 'a' },
```

Comment:
The `NavigationResetAction` type doesn't include `params` property, thus flow shows a type error. I guess this line should be removed.

Reply:
I confirm that `yarn run test` with this line shows a type error but `yarn run test` without this line shows no type error.